.gitattributes

```diff
@@ -0,0 +1,6 @@
+*       text    eol=lf
```

Comment:
Can we remove this? Why was it needed? This should be user configuration 🤔

Reply:
I find it really useful. If someone with Windows and Linux collaborate the files won't end up flipping line endings.

Reply:
I think they should configure their git to checkout and commit accordingly to their platform, so please remove.

Reply:
ok. but this not for the repositories using this hook. this is for the hook repository. so if I run a reformat or something it can easily replace the current linux line endings in this repository (which happened during creating this PR)

Reply:
I know it's for this repository. But that behaviour should not be configured here by gitattributes, it should be a global configuration by the user on their platform. I don't want to add this file for all the repositories I maintain.